labs/citizens-pulse.md

```diff
@@ -0,0 +1,68 @@
+# Lab Name
+
+Citizens Pulse
+
+# Short Description
+
+A fully distributed platform for state/city councils to know, share and feel the interests, pulse of citizens on their decisions. This is implemented with hyperledger fabric.
```

Comment:
In this section, `hyperledger fabric` should be replaced with `Hyperledger Fabric`